charts/palworld/values.yaml

```diff
@@ -1,224 +1,152 @@
----
-# -- Namespace where the resources will be created
-namespace: palworld
-# -- (dict) The server configuration
-# @notationType -- bigValue
+# Namespace where the resources will be created
+#
+namespace: palworld-server
+
+# General annotations, labels, and resources limits for the resources
+#
 server:
-  # -- Additional annotations to the resources
   annotations: {}
-  # -- Additional labels to the resources
   labels: {}
-  # -- (dict) Resources limits for the container.
   resources:
     limits:
       cpu: 4
       memory: "12Gi"
     requests:
       cpu: 4
       memory: "8Gi"
-  # -- (dict) Define some parameters for the storage volume
+
+  # Storage definitions related to the palworld-server
+  #
   storage:
-    # -- (bool) Define if it will use an existing PVC containing the installation data.
+    # -- Use an existing PVC
     external: false
-    # -- (bool) The external PVC name to use.
     externalName: ""
-    # -- Keeps helm from deleting the PVC. By default, helm does not delete pvcs.
+    # -- Keeps helm from deleting the PVC, by default helm does not delete pvcs
     preventDelete: false
-    # -- The size of the pvc storage.
     size: 12Gi
-    # -- The storage class name.
     storageClassName: ""
-  # -- (dict) Define the parameters for the server image container
-  # @notationType -- bigValue
+
+  # Docker image used for the palworld-server deployment
+  #
   image:
-    # -- Repository of the image, without the tag.
     repository: thijsvanloef/palworld-server-docker
-    # -- The tag of the image.
     tag: latest
-    # -- Define the pull policy for the server image.
     imagePullPolicy: IfNotPresent
 
-  # -- (dict) Change the ports to be mapped to the pod.
-  # If you change those, make sure to change the service.ports and server.config accordingly.
-  # @notationType -- bigValue
+  # Server-related ports
+  # Be sure your service.ports and config.ports matches this section
+  #
   ports:
-    # -- (dict) The "game" port definition.
-    # If you change this, make sure to change the service.ports.game and server.config accordingly.
-    - name: game
-      containerPort: 8211
-      protocol: UDP
-    # -- (dict) The "query" port definition .
-    # If you change this, make sure to change the service.ports.query_port and server.config accordingly.
-    - name: query
-      containerPort: 27015
-      protocol: UDP
-    # -- (dict) The "rcon" port definition .
-    # If you change this, make sure to change the service.ports.rcon and server.config accordingly.
-    - name: rcon
-      containerPort: 25575
-      protocol: UDP
-  # -- (string) Change the deployment strategy
+  - name: game
+    containerPort: 8211
+    protocol: UDP
+  - name: query
+    containerPort: 27015
+    protocol: UDP
+  - name: rcon
+    containerPort: 25575
+    protocol: UDP
```

Comment:
Isn't [rcon](https://developer.valvesoftware.com/wiki/Source_RCON_Protocol#:~:text=The%20Source%20RCON%20Protocol%20is,remote%20console%22%2C%20or%20RCON.) TCP?

Reply:
It is, that must've slipped past previous PR's

Fixed